EPRN.Portal/Services/HomeServices/ExporterAndReprocessorHomeService.cs

```diff
@@ -1,28 +1,25 @@
 ﻿using EPRN.Portal.Configuration;
 using EPRN.Portal.Resources;
+using EPRN.Portal.RESTServices.Interfaces;
 using EPRN.Portal.Services.Interfaces;
 using EPRN.Portal.ViewModels;
-using Microsoft.AspNetCore.Mvc;
+using EPRN.Portal.ViewModels.Waste;
 using Microsoft.Extensions.Options;
-using static EPRN.Common.Constants.Strings;
 
 namespace EPRN.Portal.Services
 {
-    public class ExporterAndReprocessorHomeService : BaseHomeService, IHomeService
+    public class ExporterAndReprocessorHomeService : UserBasedBaseService, IUserBasedService
     {
-        public ExporterAndReprocessorHomeService(
-            IUrlHelper urlHelper,
-            IOptions<AppConfigSettings> configSettings) : base(urlHelper, configSettings)
-        {          
+        public ExporterAndReprocessorHomeService(IOptions<AppConfigSettings> configSettings, IHttpJourneyService httpJourneyService) 
+            : base(configSettings, httpJourneyService)
+        {
         }
 
         protected override List<CardViewModel> GetCardViewModels()
         {
             var wasteCardLinks = new Dictionary<string, string>()
             {
-                { HomePageResources.HomePage_Waste_Link_RecordWaste, _urlHelper.ActionLink(
-                    Routes.Actions.Waste.RecordWaste,
-                    Routes.Controllers.Waste) },
+                { HomePageResources.HomePage_Waste_Link_RecordWaste, "#" },
```

Comment:
This still hasn't been done

Reply:
I updated this yesterday, not sure what happened to my comments.

This class/method was not part of the current story, but is from a previous one. That story is already being tested and a bug has been raised since not all links are working. It is worth noting that at the time of doing the previous story, the necessary additional pages/links had not been created. In any case, your comment above will get handled as part of the bug I mentioned: 300097